### ISSUE TITLE
Update List

### ISSUE THREAD

**Issue by dartanian300** [davidhund/styleguide-generators#60]
It seems this list hasn't been properly updated in a while. Many tools are no longer supported/in development/active.

For example:

- Stylemark has been archived
- Source JS website no longer exists
- Tapestry App lands on a 404 page

If this list isn't going to be maintained, it would be helpful to note that at the top of the `README.md`.

**Comment by davidhund** [comment]
You are right: I have not been looking at/updating this list for a long time and it would probably be a good thing to communicate this better.

Also: PR's fixing this are very welcome ;-)
